src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/FareProductHeader.tsx

```diff
@@ -1,33 +1,78 @@
-import {getTextForLanguage, useTranslation} from '@atb/translations';
+import {
+  getTextForLanguage,
+  PurchaseOverviewTexts,
+  useTranslation,
+} from '@atb/translations';
 import {StyleProp, View, ViewStyle} from 'react-native';
-import {FareProductTypeConfig} from '@atb/configuration';
+import {
+  FareProductTypeConfig,
+  PreassignedFareProduct,
+} from '@atb/configuration';
 import React, {forwardRef} from 'react';
 import {ThemeText} from '@atb/components/text';
 import {StyleSheet} from '@atb/theme';
 import {TransportationIconBoxList} from '@atb/components/icon-box';
+import {Button} from '@atb/components/button';
+import {Info} from '@atb/assets/svg/mono-icons/status';
+import {useIsTicketInformationEnabled} from '@atb/stacks-hierarchy/Root_PurchaseOverviewScreen/use-is-ticket-information-enabled';
 
 type Props = {
   fareProductTypeConfig: FareProductTypeConfig;
+  preassignedFareProduct: PreassignedFareProduct;
+  onTicketInfoButtonPress: () => void;
   style?: StyleProp<ViewStyle>;
 };
 export const FareProductHeader = forwardRef<View, Props>(
-  ({fareProductTypeConfig, style}: Props, ref) => {
-    const {language} = useTranslation();
+  (
+    {
+      fareProductTypeConfig,
+      preassignedFareProduct,
+      onTicketInfoButtonPress,
+      style,
+    }: Props,
+    ref,
+  ) => {
+    const {t, language} = useTranslation();
     const styles = useStyle();
+    const [isTicketInformationEnabled] = useIsTicketInformationEnabled();
 
     return (
-      <View style={[style, styles.header]} ref={ref} accessible={true}>
-        <TransportationIconBoxList
-          modes={fareProductTypeConfig.transportModes}
-          iconSize="normal"
-        />
-        <ThemeText
-          type="heading--medium"
-          color="background_accent_0"
-          style={{flexShrink: 1}}
-        >
-          {getTextForLanguage(fareProductTypeConfig.name, language) ?? ''}
-        </ThemeText>
+      <View style={style}>
+        <View style={styles.header} ref={ref} accessible={true}>
+          <TransportationIconBoxList
+            modes={fareProductTypeConfig.transportModes}
+            iconSize="normal"
+          />
+          <ThemeText
+            type="heading--medium"
+            color="background_accent_0"
+            style={styles.headerText}
+          >
+            {getTextForLanguage(fareProductTypeConfig.name, language) ?? ''}
+          </ThemeText>
+        </View>
+        {isTicketInformationEnabled && (
+          <View style={styles.headerSubSection}>
+            <ThemeText
+              type="body__secondary"
+              color="background_accent_0"
+              style={styles.ticketDescription}
+              numberOfLines={1}
```

Comment:
Should it support markdown here as well? In case it starts with bold or something.

Reply:
I get your point, but I'm not sure what's the best option. I've added `.replaceAll('\n', ' ')` to get the text to flow as one line. I think rendering markup here could have unknown consequences, e.g. if the markdown contains a url that's broken by the line length limit etc. If anything, I think the best option would actually be to strip the markdown markup, but I don't know if that is trivial.

Reply:
It doesn't seem like Marked har any extensions for stripping markup: https://marked.js.org/using_advanced#extensions

My two-minutes research haven't found any other small, well maintained 3rd party libraries for this either. Writing one from scratch with regexes and stuff is certainly not impossible, but perhaps more work and risk than we want for this particular use case.

A safe option could be to add a separate property to preassigned fare product, e.g. `productDescriptionShort`. We are already adding a custom property for this. What do you think, @gorandalum and @marius-at-atb ?

Reply:
I kinda agree that even if there is markdown in the text, it is not desirable to show that in this header text.
However it would be preferable imo to solve it with code. Could e.g. [strip-markdown](https://github.com/remarkjs/strip-markdown) be an option?

Reply:
As far as I can tell, strip-markdown is a plugin for Remark. We are using Marked not Remark for markdown, so I don't think it's a good idea to include Remark just to use that one plugin.

Reply:
I've created a simple `strip-markdown` function now using Marked.